docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -150,6 +151,37 @@ In the factory data set, the following formats are used:
 -   All certificates stored in factory data are provided in the
     [X.509](https://www.itu.int/rec/T-REC-X.509-201910-I/en) format.
 
+#### Appearance field description
+
+The `appearance` field in the factory data set describes the device's visible
+appearance.
+
+-   `finish` - A string name that indicates the visible exterior finish of the
+    product. It refers to the `ProductFinishEnum` enum, and currently, you can
+    choose one of the following names:
+
+|    Name    | Enum value |
+| :--------: | :--------: |
+|  `matte`   |     0      |
+|  `satin`   |     1      |
+| `polished` |     2      |
+|  `rugged`  |     3      |
+|  `fabric`  |     4      |
+|  `other`   |    255     |
+
+-   `primary_color` - A string name that represents the RGB color space of the
+    device's case color, which is the most representative. It refers to the
+    `ColorEnum` enum, and currently, you can choose one of the following names:
+
+(Enum value) color name (`RGB value`)
+
+| (0) $$\color{black} \color{black}{black}$$ (`#000000`) | (1) $$\color{#000080}{navy}$$ (`#000080`)    | (2) $$\color{#008000}{green}$$ (`#008000`)    | (3) $$\color{#008080}{teal}$$ (`#008080`)                                       | (4) $$\color{#800080}{maroon}$$ (`#800080`)             |
+| ------------------------------------------------------ | -------------------------------------------- | --------------------------------------------- | ------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| (5) $$\color{#800080}{purple}$$ (`#800080`)            | (6) $$\color{#808000}{olive}$$ (`#800080`)   | (7) $$\color{#808080}{gray}$$ (`#800080`)     | (8) $$\color{blue}{blue}$$ (`#0000FF`)                                          | (9) $$\color{lime}{lime}$$ (`#00FF00`)                  |
+| (10) $$\color{aqua}{aqua}$$ (`#00FFFF`)                | (11) $$\color{red}{red}$$ (`#FF0000`)        | (12) $$\color{fuchsia}{fuchsia}$$ (`#FF00FF`) | (13) $$\color{yellow}{yellow}$$ (`#FFFF00`)                                     | (14) $$\color{white} \color{white}{white}$$ (`#800080`) |
+| (15) $$\color{#727472}{nickel}$$ (`#727472`)           | (16) $$\color{#a8a9ad}{chrome}$$ (`#a8a9ad`) | (17) $$\color{#E1C16E}{brass}$$ (`#E1C16E`)   | (18) $$\color{#E1C16E}{co} \color{orange}{p} \color{brown}{pe} \color{red}{r}$$ | (19) $$\color{#C0C0C0}{silver}$$ (`#C0C0C0`)            |
```

Comment:
Why is copper not a single color?

Reply:
Hmm, it can be one color, but I found that it can be also a gradient of colors. But ok, after rethinking, it would be better to have it here as one color. :)